File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int bricks);

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    for (int row = 0; row < height; row++)
    {
        // Build the spaces of the left side of the pyramid
        for (int spaces = 0; spaces < height - row - 1; spaces++)
        {
            printf(" ");
        }
        // Build the hashes of the left side of the pyramid
        for (int hash = 0; hash <= row; hash++)
        {
            printf("#");
        }

        // Add the two spaces in between the sides of the pyramid
        printf("  ");

        // Build the hashes of the right side of the pyramid
        for (int hash = 0; hash <= row; hash++)
        {
            printf("#");
        }

        // Break line for each row
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
def main():
    while True:
        try:
            # Ask for the height of pyramid
            height = int(input("Height: "))

            # Keep asking for height if less than 2
            if height > 0 and height < 9:
                break
        # Exception if user gives non-integer value
        except ValueError:
            print("Please, provide an integer value")

    for row in range(height):
        # Build the spaces of the left side of the pyramid
        for spaces in range(height - row - 1):
            print(" ", end="")

        # Build the hashes of the left side of the pyramid
        for hash in range(row + 1):
            print("#", end="")

        # Add two spaces in between the sides of the pyramid
        print("  ", end="")

        # Build the hashes of the right side of the pyramid
        for hash in range(row + 1):
            print("#", end="")

        # Break line of each row
        print()


if __name__ == "__main__":
    main()
